database/dbHelpers.js

```diff
@@ -0,0 +1,21 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile.js');
+
+const db = knex(knexConfig.development);
+
+module.exports = {
+// all the methods needed to be available someplace else 
+
+get: (users) => {
+    return db('users').select('id', 'username')
+},
+
+insert: (user) => {
+    return db('users').insert(user);
+},
+
+findByUsername: (username) => {
+    return db('users').where('username', username);
```

Comment:
You could use an object here to make the argument for your `where` a tad shorter:
```javascript
return db('users').where({ username });
```